src/client/initialize.ts

```diff
@@ -135,7 +136,16 @@ export function initialize<T = {}>(): PluginInstance<T> {
       ) {
         void execPromise('wb:plugin:selection:set', id, elementId, selection);
       },
+      triggerAction(id: string) {
+        if (!registeredActionTriggers.has(id)) return;
```

Comment:
Hmm, I think this should throw with an error that indicates that the provided Id is not a valid action trigger.